package.json

```diff
@@ -73,7 +73,7 @@
     "restart": "npm-run-all build start",
     "transpile-tests": "tsc -p tests",
     "test-system": "node scripts/bin/mock_client",
-    "test": "jest --forceExit --runInBand --detectOpenHandles"
+    "test": "jest --forceExit --runInBand"
```

Comment:
why?

Reply:
@liorama 
it sounds good, if i was aware of this flag i would have added it to begin with, but i would also like to know the reason it was added

Reply:
He removed it. This flag is good for when jest has problems closing. If it doesn't than it should be benign to have it there (worst case, we get a warning sometimes but that's it) on the other hand if do run into a build stuck on jest not closing I prefer having this there so we don't have to push a commit=wait for the build just to see the log, but maybe I'm missing something, hence the comment :)